Source/WebCore/style/values/text-decoration/StyleTextDecorationLine.h

```diff
@@ -131,7 +132,11 @@ struct TextDecorationLine {
         return WTF::switchOn(m_value, std::forward<F>(f)...);
     }
 
+#if !COMPILER(GCC) || GCC_VERSION_AT_LEAST(13, 0, 0)
     constexpr explicit operator bool() const { return !isNone(); }
```

Comment:
Funny that this builds -- I would have assumed you have to explicitly mark `isNone()` as `constexpr` for this to compile. but since WTF::holdsAlternative() is constexpr, I'd just mark all is\* methods as constexpr.